pkg/solana/logpoller/orm_test.go

```diff
@@ -0,0 +1,135 @@
+//go:build db_tests
+
+package logpoller
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/google/uuid"
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/lib/pq"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/pg"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+	"github.com/stretchr/testify/require"
+)
+
+// NOTE: at the moment it's not possible to run all db tests at once. This issue will be addressed separately
+
+func TestLogPollerFilters(t *testing.T) {
+	lggr := logger.Test(t)
+	dbURL, ok := os.LookupEnv("CL_DATABASE_URL")
+	require.True(t, ok, "CL_DATABASE_URL must be set")
+	chainID := uuid.NewString()
+	dbx := pg.NewSqlxDB(t, dbURL)
```

Comment:
I'm really confused... I have a PR open to move `NewSqlDB` and some other things over from `chainlink` to `chainlink-common`, but as far as I know it hasn't been merged yet. 🤔

I wonder if someone else moved it over already.